admin/services/data-access/access-arrangements.data.service.js

```diff
@@ -5,6 +5,19 @@ const sqlService = require('./sql.service')
 const accessArrangementsDataService = {}
 const accessArrangementCodes = {}
 
+/**
+ * Constants to describe the access arrangements codes
+ */
+accessArrangementsDataService.CODES = Object.freeze({
```

Comment:
That's indeed very useful.
@jon-shipley @danwilliams  we should follow this pattern in the other areas as well.